antsibull.cfg

```diff
@@ -1,17 +1,21 @@
 ansible_base_url = https://github.com/ansible/ansible/
+# set breadcrumbs to false to turn off breadcrumbs at top of each html page
 breadcrumbs = True
+# set use_html_blobs to True generate raw html plugin tables instead of rst
 use_html_blobs = False
 doc_parsing_backend = ansible-internal
 chunksize = 4096
 galaxy_url = https://galaxy.ansible.com/
 indexes = True
 process_max = none
```

Comment:
```suggestion
# Maximum number of worker processes for parallel operations.
# It may be None to mean, use all available CPU cores:
process_max = none
```